Schedule erika polanco

Rebeccals opened the pull request:
Updated branch with the new changes from main. We finished the schedule page. Let us know if changes need to be made. I also updated the accordion to have a transition when clicked.

chillwafflez reviewed (changes requested):
Everything looks great! One small thing is the background of the Schedule page also seems to zoom in/change height when you click on the dropdown for each of the days. See if you could fix this so that clicking on the dropdown for each day doesn't affect the styling of the background. No worries if you can't fix it, we can also take a look at it.

Rebeccals commented:
I updated the schedule page with the changes you stated, but I also added the countdown timer changes with this branch. I hope it's Ok. I got the timer to look more like the design.

Rebeccals commented:
Updated the branch again. I think the conflicts are resolved.
